e2e/node.go

```diff
@@ -26,29 +26,38 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
+	testutils "k8s.io/kubernetes/test/utils"
 )
 
-func createNodeLabel(f *framework.Framework, labelKey, labelValue string) error {
+func addLabelsToNodes(f *framework.Framework, labels map[string]string) error {
```

Comment:
```suggestion
func addNodeLabels(f *framework.Framework, labels map[string]string) error {
```

Reply:
addLabelsToNodes seems more opt to me, same is used in Kuberentes util as well.

Reply:
thought using `deleteNodeLabels` and `addNodeLabels` would be a good idea.

Reply:
followed the format from the kubernetes package as an example, lets see what others think, based on the will change this if required.